docs/source/changelog.rst

```diff
@@ -8,6 +8,7 @@ Changelog
 - The dataframe passed to ```.to_gbq(...., if_exists='append')``` needs to contain only a subset of the fields in the BigQuery schema. (:issue:`24`)
 - Use the `google-auth <https://google-auth.readthedocs.io/en/latest/>`__ library for authentication because oauth2client is deprecated. (:issue:`39`)
 - ``read_gbq`` now has a ``auth_local_webserver`` boolean argument for controlling whether to use web server or console flow when getting user credentials. Replaces `--noauth_local_webserver` command line argument (:issue:`35`)
+- Add support to replace partitions in date-partitioned tables (:issue:`47`)
```

Comment:
can you be slightly more verbose and include the link: [date-partitioned tables](https://cloud.google.com/bigquery/docs/partitioned-tables). any addition to the main docs about this? (or enhancements to the doc-string)?